components.js

```diff
@@ -197,6 +197,10 @@ var components = {
 			"title": "Eiffel",
 			"owner": "Conaclos"
 		},
+		"elisp": {
+			"title": "Elisp",
+			"owner": "JuanCaicedo"
```

Comment:
I'm not sure if I should be the owner, since I didn't develop this. Totally willing to defer to @akirak if they would like to take it over 😀

Reply:
Thanks for this pull request. 

I don't mind your being owner of the module, but this plugin is experimental, and there are some corner cases where it doesn't render well, like in [this example](https://jingsi.space/post/2017/04/05/organizing-a-complex-directory-for-emacs-org-mode-and-deft/). It's doing complex things, and I don't know how to fix this bug. 

If you still want to accept this patch, you or whoever can be the owner. I will mention this merge in the README of my repo, but I won't be responsible for the plugin.

Reply:
@akirak has a point, here. In Prism, we do expect the component's "owner" to do some kind of maintenance if issues regarding this component arise.
So you should probably keep it with your ownership, @JuanCaicedo.